spec/controllers/works/default_rails_actions_spec.rb

```diff
@@ -577,14 +577,21 @@ def call_with_params(params)
       before { run_all_indexing_jobs }
 
       context "as a guest" do
-        it "renders the empty collected form" do
-          get :collected
+        it "renders the collected form" do
+          get :collected, params: { user_id: collected_user.login }
           expect(response).to render_template("collected")
         end
 
-        it "does NOT return any works if no user is set" do
-          get :collected
-          expect(assigns(:works)).to be_nil
+        it "returns not found error if user does not exist" do
+          expect {
+            get :collected, params: { user_id: "dummyuser" }
+          }.to raise_error(ActiveRecord::RecordNotFound)
+        end
```

Comment:
This test and the one below it are nested inside of the `with restricted works` context, but there's nothing about these tests that is specific to users with restricted works. (I think they should apply regardless of which properties the works have.) It might be good to move them out of this context block, possibly into their own context block(s).

Reply:
Yeah, that's what I was thinking too.  I'll just put them top-level under the describe `collected` block.